CMakeLists.txt

```diff
@@ -0,0 +1,2075 @@
+option(GPERFTOOLS_ENABLED "Google PerfTools libtcmalloc linkage" OFF)
+option(GPERFTOOLSPROFILER_ENABLED "Google PerfTools libprofiler linkage" OFF)
+if(GPERFTOOLS_ENABLED OR GPERFTOOLSPROFILER_ENABLED)
+  find_package(GPerfTools REQUIRED)
+  if(GPERFTOOLS_ENABLED)
+    target_link_libraries(mixxx-lib PUBLIC GPerfTools::tcmalloc)
+  endif()
+  if(PERFTOOLSPROFILER_ENABLED)
+    target_link_libraries(mixxx-lib PUBLIC GPerfTools::profiler)
+  endif()
+endif()
+
+# HSS1394 MIDI device
+find_package(HSS1394)
+cmake_dependent_option(HSS1394_ENABLED "HSS1394 MIDI device support" ON "HSS1394_FOUND;WIN32 OR APPLE" OFF)
+if(HSS1394_ENABLED)
+  target_sources(mixxx-lib PRIVATE
+    src/controllers/midi/hss1394controller.cpp
+    src/controllers/midi/hss1394enumerator.cpp
+  )
+  target_compile_definitions(mixxx-lib PUBLIC __HSS1394__)
+  if(WIN32 OR APPLE)
+    if(NOT HSS1394_FOUND)
+      message(FATAL_ERROR "HSS1394 MIDI device support requires the libhss1394 and its development headers.")
+    endif()
+    target_link_libraries(mixxx-lib PUBLIC HSS1394::HSS1394)
+  endif()
+endif()
+
+# Lilv (LV2)
+find_package(Lilv)
+cmake_dependent_option(LILV_ENABLED "Lilv (LV2) support" ON "Lilv_FOUND" OFF)
+if(LILV_ENABLED)
+  if(NOT Lilv_FOUND)
+    message(FATAL_ERROR "Lilv (LV2) support requires the liblilv-0 and its development headers.")
+  endif()
+  target_sources(mixxx-lib PRIVATE
+    src/effects/lv2/lv2backend.cpp
+    src/effects/lv2/lv2effectprocessor.cpp
+    src/effects/lv2/lv2manifest.cpp
+    src/preferences/dialog/dlgpreflv2.cpp
+  )
+  target_compile_definitions(mixxx-lib PUBLIC __LILV__)
+  target_link_libraries(mixxx-lib PUBLIC Lilv::Lilv)
+endif()
+
+# Live Broadcasting (Shoutcast)
+find_package(Shout)
+cmake_dependent_option(BROADCAST_ENABLED "Live Broadcasting (Shoutcast) support" ON "Shout_FOUND" OFF)
+if(BROADCAST_ENABLED)
+  if(NOT Shout_FOUND)
+      message(FATAL_ERROR "Live Broadcasting support requires the libshout and its development headers.")
+  endif()
+  target_sources(mixxx-lib PRIVATE
+    src/preferences/dialog/dlgprefbroadcastdlg.ui
+    src/preferences/dialog/dlgprefbroadcast.cpp
+    src/broadcast/broadcastmanager.cpp
+    src/engine/sidechain/shoutconnection.cpp
+  )
+  target_compile_definitions(mixxx-lib PUBLIC __BROADCAST__)
+  target_link_libraries(mixxx-lib PUBLIC Shout::Shout)
+endif()
+
+# Locale Aware Compare for SQLite
+find_package(SQLite3)
+cmake_dependent_option(LOCALECOMPARE_ENABLED "Locale Aware Compare support for SQLite" ON "SQLite3_FOUND" OFF)
+if(LOCALECOMPARE_ENABLED)
+  if(NOT SQLite3_FOUND)
+    message(FATAL_ERROR "Locale Aware Compare for SQLite requires libsqlite and its development headers.")
+  endif()
+  target_compile_definitions(mixxx-lib PUBLIC __SQLITE3__)
+  target_include_directories(mixxx-lib SYSTEM PRIVATE ${SQLite3_INCLUDE_DIRS})
+  target_link_libraries(mixxx-lib PUBLIC ${SQLite3_LIBRARIES})
+endif()
+
+# Opus (RFC 6716)
+find_package(Opus)
+find_package(Celt)
+find_package(Silk)
```

Comment:
We can move this into the "if(WIN32 AND STATIC_DEPS)" branch.

Reply:
Done.